docs/source/substitutions.sub

```diff
@@ -79,15 +79,15 @@
 
 .. |BaseToolDeclaration| replace:: :py:class:`~ecpy.measure.base_tool.BaseToolDeclaration`
 
-.. |BasePreHook| replace:: :py:class:`~ecpy.measure.hooks.base_hooks.BasePreHook`
+.. |BasePreExecutionHook| replace:: :py:class:`~ecpy.measure.hooks.base_hooks.BasePreExecutionHook`
```

Comment:
This confirms my first comment. Could you try to build the docs using 'make html' from the docs directory. You need sphinx installed. To see if there are any errors.